src/ROS_nav_c++/robot_setup_tf/package.xml

```diff
@@ -58,8 +58,8 @@
   <exec_depend>geometry_msgs</exec_depend>
   <exec_depend>roscpp</exec_depend>
   <exec_depend>tf</exec_depend>
-  <depend package="tf"/>
```

Comment:
Will these packages be re-enabled in the future?

Reply:
Yes, they are part of the future integration for the navigation stack. This might change tho

Reply:
Understood! Looks good to keep.